### PR TITLE
Add support for removing users/groups from a project and allow to check for groups with permissions to modify a PR

### PR DESCRIPTION
**Pull request by nforro**
Related to https://github.com/packit/packit/issues/2078.

Merge before https://github.com/packit/packit/pull/2112.

### CODE REVIEW

File: ogr/abstract.py [hunk 1]
```diff
@@ -1542,6 +1558,15 @@ def add_group(self, group: str, access_level: AccessLevel) -> None:
...
         raise NotImplementedError
 
+    def remove_group(self, group: str) -> None:
+        """
+        Remove group from project.
+
+        Args:
+            group: Name of the group.
+        """
+        raise NotImplementedError
```
Comment (lbarcziova) [inline ogr/abstract.py:1568]:
```suggestion
        raise NotImplementedError()
```

**Review by lbarcziova — APPROVED** [review]
LGTM

File: ogr/abstract.py (hunk 1 above)
Comment (nforro) [reply ogr/abstract.py:1568]: Ok, let me fix also the one above :slightly_smiling_face:

**Comment by nforro** [comment]
> could you please also update [COMPATIBILITY.md](https://github.com/packit/ogr/blob/main/COMPATIBILITY.md)?

It looks like `add_group` is also missing from the document (not supported on GitHub and GitLab), I'll add it as well.

File: COMPATIBILITY.md [hunk 2]
```diff
@@ -51,12 +51,16 @@ In case you find any error, please [create a new issue](https://github.com/packi
...
+|                             | GitHub | GitLab |         Pagure          |
+| --------------------------- | :----: | :----: | :---------------------: |
+| `change_token`              |   ✘    |   ✔   |           ✔            |
+| `get_release`               |   ✔   |   ✔   |            ✘            |
+| `get_latest_release`        |   ✔   |   ✔   |            ✘            |
+| `is_private`                |   ✔   |   ✔   | ✘ (may not be accurate) |
+| `remove_user`               |   ✘    |   ✘    |           ✔            |
+| `add_group`                 |   ✘    |   ✘    |           ✔            |
+| `remove_group`              |   ✘    |   ✘    |           ✔            |
+| `which_groups_can_merge_pr` |   ✘    |   ✘    |           ✔            |
```
Comment (lbarcziova) [inline COMPATIBILITY.md:64]: thanks!

**Pull request merged**